src/clj/harja/palvelin/tyokalut/lukot.clj

```diff
@@ -31,12 +31,9 @@
   ([db tunniste toiminto-fn vanhenemisaika]
    (if (lukko/aseta-lukko? db tunniste vanhenemisaika)
      (do
-       (log/info (format "Lukkoa: %s ei ole asetettu. Voidaan ajaa toiminto." tunniste))
```

Comment:
Mutta nämä. Näitä on lisätty lukkojen toiminnan varmistamiseksi. En ole varma, että uskaltaako näitä poistaa. anyone?

Reply:
Joo hyvä pointti ja samaa itsekin mietin. Näiden avulla varsinkin nuo kerran päivässä toistuvat yölliset ajot on voitu todentaa tapahtuvan vain kerran. Nyt ne toimivat, mutta jos loki poistetaan ja koodia muuttuu, jää se vähän usvaan. Ehkä voisi tutkia syntyykö kaikista päivittäin toistuvista ajoista jokin muu lokirivi, mikä minusta riittäisi. Sittenhän nähdään että sellainen syntyy vain yhdeltä nodelta. Tällöin tämän rivin voisi poistaa.

Reply:
Tuntuisko Jere pahalta, jos kuitenkin poistaisi vain tuon yhden logituksen ja tekisi sen virvelöinnin asetuksiin, että sitä ajettaisiin lokaalisti harvemmin?

Reply:
Tämä kuulostaa hyvältä omaan korvaan. Palaan tähän huomenna jos ei muuta tule vastaan

Reply:
Tämä poistettu, kuittausten monitoroinnin viive nostettu kymmeneen minuuttiin jos ollaan kehitysmoodissa.